appender/elasticsearch/src/main/java/org/apache/karaf/decanter/appender/elasticsearch/ElasticsearchAppender.java

```diff
@@ -165,7 +165,7 @@ private void send(Event event) throws Exception {
         String endpoint = String.format("/%s/%s", indexName, getValue(config, INDEX_TYPE_PROPERTY, INDEX_TYPE_DEFAULT));
         HttpEntity request = new NStringEntity(jsonSt, ContentType.APPLICATION_JSON);
 
-        client.performRequest("POST", endpoint, Collections.singletonMap("refresh", "true"), request);
+        client.performRequest("POST", endpoint, null, request);
```

Comment:
Maybe (not sure for this PR specifically), it makes sense to do:

```
        Request request = new Request("POST", endpoint);
        request.setEntity(entity);
        Response response = client.performRequest(request);
        if (response.getStatusLine().getStatusCode() != HttpStatus.SC_OK) {
            LOGGER.warn("Can't append in Elasticsearch ({}): {}", response.getStatusLine().getStatusCode(), response.getStatusLine().getReasonPhrase());
        }
```

Reply:
By the way, this code comes from my Elasticsearch 7 support branch.

Reply:
@jbonofre do you want me to add this or we wait for you ES7 branch?

Reply:
@fpapon don't worry, it's already part of the ES7 branch, so, we are good.